rm_reporting/controllers/case_controller.py

```diff
@@ -25,8 +26,13 @@ def get_case_data(collection_exercise_id: str) -> list:
         "ORDER BY sample_unit_ref, status"
     )
 
-    with case_engine.begin() as conn:
-        return conn.execute(case_business_ids_query, {"collection_exercise_id": collection_exercise_id}).all()
+    try:
+        with case_engine.begin() as conn:
+            result = conn.execute(case_business_ids_query, {"collection_exercise_id": collection_exercise_id}).all()
+    except SQLAlchemyError:
+        logger.error("Failed to get case data", collection_exercise_id=collection_exercise_id)
+        return
```

Comment:
These would return a None which cause an error further up in the request that is not handled - we should `use return make_response(...)` to return straight to rOps with a code and information about what happened or add NoneType error handling where there is the chance for a NoneType

Reply:
I ended up changing the `except:` block so that the `statement` parameter associated with the error (which contains the query) is replaced with a blank string before the error is raised, so that the query does not appear in the logs.

Reply:
As Matthew said below we shouldn't be overriding elements of the error rather extract the bits we need from it eg `e.__class__.__name__`

Reply:
I have set it so that the logging no longer displays anything about the error (because doing so would print out the SQL query). The `statement` parameter is no longer modified.